### ISSUE TITLE
V7 - `on_conflict:` no longer accepts a Hash

### ISSUE THREAD

**Issue by Matchlighter** [mhenrixon/sidekiq-unique-jobs#495]
**Describe the bug**
Version 7 Beta 14 does not work when given a Hash for `on_conflict:`.
I'm running on Ruby 2.6.3.

**Expected behavior**
I can provide a Hash to `on_conflict:`

**Current behavior**
When the Conflict occurs, the following Error is thrown:
```
NoMethodError: undefined method `call' for {"client"=>"log", "server"=>"reschedule"}:Hash
<truncated>/sidekiq-unique-jobs-7.0.0.beta14/lib/sidekiq_unique_jobs/lock_config.rb:116:in `on_server_conflict'
<truncated>/sidekiq-unique-jobs-7.0.0.beta14/lib/sidekiq_unique_jobs/lock/base_lock.rb:148:in `server_strategy'
<truncated>/sidekiq-unique-jobs-7.0.0.beta14/lib/sidekiq_unique_jobs/lock/while_executing.rb:40:in `block in execute'
```

**Worker class**

```ruby
class SyncBlueprintWorker
  include Sidekiq::Worker

  sidekiq_options(
    lock: :until_and_while_executing,
    on_conflict: { client: :log, server: :reschedule },
  )
  ...
end
```

**Additional Context**
I've boiled it down to these snippets `on_conflict&.(:[], :server)` & `on_conflict&.(:[], :client)`. I don't understand the need for the `&.(:[], )` syntax (as opposed to just `on_conflict[:client]`) - there's already a guard ensuring it's a Hash.

**Comment by mhenrixon** [comment]
Thanks for the report @Matchlighter, this will be fixed in the next version. I'll hopefully release a new version today.

**Comment by mhenrixon** [comment]
Released as v7.0.15
